addons/project/migrations/9.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -1,6 +1,6 @@
 ---Fields in module 'project'---
-project      / project.project          / date (date)                   : NEW
-project      / project.project          / date_start (date)             : NEW
+project      / project.project          / date (date)                   : DONE: post-migration: Obtained from analytic account if any.
+project      / project.project          / date_start (date)             : DONE: post-migration: Obtained from analytic account if any.
```

Comment:
The way we do it is not altering the lines, but adding a comment by blocks this way:
```suggestion
project      / project.project          / date (date)                   : NEW
project      / project.project          / date_start (date)             : NEW
# DONE: post-migration: Obtained from analytic account if any.
```